x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_results.tsx

```diff
@@ -157,6 +193,7 @@ export const LogRateAnalysisResults: FC<LogRateAnalysisResultsProps> = ({
   );
   const [shouldStart, setShouldStart] = useState(false);
   const [toggleIdSelected, setToggleIdSelected] = useState(resultsGroupedOffId);
+  const [skippedColumns, setSkippedColumns] = useState<string[]>(['p-value']);
```

Comment:
It would be nice if this could be `ColumnNames[]` rather than string? It'll probably mean changing all places where they are stored as a string array, but it'd be nice to use the types to enforce that only the correct values are used.